be/src/service/test.service.ts

```diff
@@ -3,21 +3,23 @@ import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
 
 import { TestModel } from '../models/test.model';
-import { TestDTO } from './test.dto'
+import { TestDTO } from './test.dto';
 
 @Injectable()
 export class TestService {
-    constructor(@InjectRepository(TestModel) private testRepository: Repository<TestModel>,) {}
+  constructor(
+    @InjectRepository(TestModel) private testRepository: Repository<TestModel>,
+  ) {}
 
-    create(testDTO: TestDTO): Promise<TestModel> {
-        return this.testRepository.save(testDTO);
-    }
+  create(testDTO: TestDTO): Promise<TestModel> {
+    return this.testRepository.save(testDTO);
+  }
 
-    findAll(): Promise<TestModel[]> {
-        return this.testRepository.find();
-    }
+  findAll(): Promise<TestModel[]> {
+    return this.testRepository.find();
+  }
 
-    findById(id: string): Promise<TestModel> {
-        return this.testRepository.findOne(id);
-    }
-}
\ No newline at end of file
+  findById(id: string): Promise<TestModel> {
```

Comment:
what if can't be found will it still return `Promise<TestModel>`?

Reply:
![image](https://user-images.githubusercontent.com/33936483/115609806-af30e500-a2f0-11eb-9386-dfff5d89d47c.png)
The return type of the `FindOne` is `Promise<Entity | undefined>` so it might return a promise with no entity. I'm not sure why ts is allowing to set our `findById` function like this. I'll look into this.

Reply:
@Orzelius take a look other functions as well while u are at it. Might also need to add js docs for thrown exceptions.

Reply:
@Kasparsu Let's not complicate this pr of bootstrapping the be any further. I'd like to get it done by Friday and then we can start polishing it.

> Might also need to add js docs for thrown exceptions.

I've never, not once in my lifetime seen jsDocs used in a ts project. This case here seems weird to me as ts should 100% not be happy with the return types of `findById` function, I think it's a config issue or some nestjs thing I'm not aware of. Will research,

Reply:
@Orzelius sure you are right, was just a suggestion. Typescript is not my forte.